plugins/package-managers/node/src/main/kotlin/Npm.kt

```diff
@@ -28,6 +28,7 @@ import org.apache.logging.log4j.kotlin.logger
 
 import org.ossreviewtoolkit.analyzer.AbstractPackageManagerFactory
 import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.analyzer.PackageManager.Companion.processPackageVcs
```

Comment:
Commit message: The part "from other child classes of `Npm`, once they get refactored to not inherit from `Npm` anymore" reads a bit funny because if they don't inherit anymore, they're also no child classes. Maybe just add "from *current* child classes"?